go.mod

```diff
@@ -54,6 +54,5 @@ require (
 	golang.org/x/net v0.0.0-20190311183353-d8887717615a
 	google.golang.org/grpc v1.19.0
 	gopkg.in/DATA-DOG/go-sqlmock.v1 v1.3.0
-	gopkg.in/mgo.v2 v2.0.0-20160818020120-3f83fa500528
```

Comment:
we don't use mgo any more??

Reply:
somehow, the `go mod tidy` command delted mongo. Added it back by running `go build` again.